code/mobile/android/PhoneVR/app/src/noGvr/AndroidManifest.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <application>
+        <activity
+           android:name=".ALVRActivity"
```

Comment:
This is too direct, if i understood the noGvr version. 

I was unable to test this but seeing the code it seems like there is only one activity which will launch directly into VRActivity. Changing contents of ALVR Lobby room is hard, so InitActivity must be there in both versions. We need to introduce the user to PhoneVR application, some docs, some version, some help links, some other information or where to download the streamer etc. Discord server link or something like that. We cant expect users to know everything perfectly and rather we should guide him to ALVRActivity.

Do take the case where the App is installed from playstore or FOSS-store.

Reply:
Will change it, but might take a while

Reply:
DOne